org.jasp-stats.JASP.json

```diff
@@ -0,0 +1,5548 @@
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/stringi_1.3.1.tar.gz",
+					"sha256": "32df663bb6e9527e1ac265eec2116d26f7b7e62ea5ae7cc5de217cbb8defc362"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib",
+				"env": {
+					"GIT_DISCOVERY_ACROSS_FILESYSTEM": "true",
+					"R_HOME": "/app/lib64/R/",
+					"PREFIX": "/app"
+				}
+			},
+			"build-commands": [ "R CMD INSTALL ." ]
+		},
+
+		{
+			"name": "stringr",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/stringr_1.4.0.tar.gz",
+					"sha256": "87604d2d3a9ad8fd68444ce0865b59e2ffbdb548a38d6634796bbd83eeb931dd"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib",
+				"env": {
+					"GIT_DISCOVERY_ACROSS_FILESYSTEM": "true",
+					"R_HOME": "/app/lib64/R/",
+					"PREFIX": "/app"
+				}
+			},
+			"build-commands": [ "R CMD INSTALL ." ]
+		},
+
+		{
+			"name": "SuppDists",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/SuppDists_1.1-9.4.tar.gz",
+					"sha256": "fcb571150af66b95dcf0627298c54f7813671d60521a00ed157f63fc2247ddb9"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib",
+				"env": {
+					"GIT_DISCOVERY_ACROSS_FILESYSTEM": "true",
+					"R_HOME": "/app/lib64/R/",
+					"PREFIX": "/app"
+				}
+			},
+			"build-commands": [ "R CMD INSTALL ." ]
+		},
+
+		{
+			"name": "sys",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/sys_3.0.tar.gz",
+					"sha256": "b05f576923b7581976fce19b707d428dc8315fcd3805255fe9e3566c8a0b18bc"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib",
+				"env": {
+					"GIT_DISCOVERY_ACROSS_FILESYSTEM": "true",
+					"R_HOME": "/app/lib64/R/",
+					"PREFIX": "/app"
+				}
+			},
+			"build-commands": [ "R CMD INSTALL ." ]
+		},
+
+		{
+			"name": "TH.data",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/TH.data_1.0-10.tar.gz",
+					"sha256": "618a1c67a30536d54b1e48ba3af46a6edcd6c2abef17935b5d4ba526a43aff55"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib",
+				"env": {
+					"GIT_DISCOVERY_ACROSS_FILESYSTEM": "true",
+					"R_HOME": "/app/lib64/R/",
+					"PREFIX": "/app"
+				}
+			},
+			"build-commands": [ "R CMD INSTALL ." ]
+		},
+
+		{
+			"name": "truncnorm",
+			"buildsystem": "simple",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://static.jasp-stats.org/RPkgs/truncnorm_1.0-8.tar.gz",
+					"sha256": "49564e8d87063cf9610201fbc833859ed01935cc0581b9e21c42a0d21a47c87e"
+				}
+			],
+			"build-options": {
+				"append-ld-library-path": "/app/lib;/app/lib64/R/lib
```

Comment:
Those build options are shared between a lot of modules; it would be better to put those outside of modules so they are shared with all the modules.

Reply:
Is that easy to do?

Right now it is generated by a script anyway, so most would get replaced easily.

Reply:
Yes.  Just copy paste the build-options outside of modules and remove it from each module.